### PR TITLE
when parsing observationCriteria, the last code/value found is the drc

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: test/unit/measure-loader/cql_loader_test.rb [hunk 1]
```diff
@@ -263,6 +263,23 @@ def test_5_4_CQL_measure
     end
   end
 
+  def test_5_4_CQL_measure_with_dcr
+    VCR.use_cassette("measure__test_5_4_CQL_measure_with_dcr", @vcr_options) do
```
Comment (mayerm94) [inline test/unit/measure-loader/cql_loader_test.rb]: Should be "measure__test_5_4_CQL_measure_with_drc" -- will require a re-record of the cassette. Test title should also say "drc" instead of "dcr"

File: test/unit/measure-loader/cql_loader_test.rb [hunk 2]
```diff
@@ -263,6 +263,23 @@ def test_5_4_CQL_measure
...
+      loader = Measures::CqlLoader.new(measure_file, @measure_details, value_set_loader)
+      measures = loader.extract_measures
+      measure = measures[0]
+
+      assert_equal "Childhood Immunization Status", measure.title
+      assert_equal "40280382-6A17-9FBF-016A-513598AF15AD", measure.hqmf_id
+      assert_equal "B2802B7A-3580-4BE8-9458-921AEA62B78C", measure.hqmf_set_id
+      assert_equal 1, measure.population_sets.size
+      assert_equal 4, measure.population_criteria.keys.count
+      assert_equal 3, measure.cql_libraries.size
```
Comment (mayerm94) [inline test/unit/measure-loader/cql_loader_test.rb]: Should add an assertion for the valueset that is now brought in via the new parsing path

**Comment by hossenlopp** [comment]
Approved. But this does need to be updated with master.
